src/components/PdfProvider/PdfProvider.tsx

```diff
@@ -21,17 +21,36 @@ export interface IPdfContext extends IPdfServices {
   viewer: pdfJsViewer.PDFViewer | undefined;
   setViewer: Dispatch<SetStateAction<pdfJsViewer.PDFViewer | undefined>>;
   scale: number;
+  lightThemeEnabled: boolean;
+  setLightThemeEnabled: Dispatch<SetStateAction<boolean>>;
 }
 
 interface IPdfProviderProps {
   pdfUrl: string;
   children: ReactNode;
 }
 
+const THEME_LOCAL_STORAGE_KEY = "light-theme-enabled";
+
+function loadThemeSettingFromLocalStorage() {
+  const localStorageValue = window.localStorage.getItem(THEME_LOCAL_STORAGE_KEY) ?? "false";
+
+  return localStorageValue.toLowerCase() === "true";
+}
+
+function saveThemeSettingToLocalStorage(value: boolean) {
+  try {
+    window.localStorage.setItem(THEME_LOCAL_STORAGE_KEY, value.toString());
+  } catch (e) {
+    alert(`Unable to save theme setting: ${e}`);
```

Comment:
```suggestion
    console.error(`Unable to save theme setting: ${e}`);
```
That's not a critical action that would justify interrupting user's flow.

Reply:
correct, dont know how that got there